airflow/config_templates/default_airflow.cfg

```diff
@@ -364,12 +368,12 @@ authenticate = False
 
 [ldap]
 # set this to ldaps://<your.ldap.server>:<port>
-uri = 
+uri =
 user_filter = objectClass=*
 user_name_attr = uid
 group_member_attr = memberOf
-superuser_filter = 
-data_profiler_filter = 
```

Comment:
Unrelated changes, but I prefer trimmed lines :-)

Reply:
Agree, I'm tempted to leave it in :)